Vizzuality/heco-invest
# Add form/SDGs component

This PR adds an SDGs form component that allows the user to select/deselect SDGs. 
The component was built based on our existing Tag/TagGroup form components. 

**Changes:**

- Added a new `forms/SDGs` component
  _to allow the user to select SDGs_  
- Added SDGs mockup data
  _to ease working with them in the future_    
- Fixed a transparency issue in our current SDG images  
  _Due to how `forms/SDGs` is built, the transparency issue would cause a gray border to show around some of the images_

## Testing instructions

Verify that: 

1. Selecting and deselecting SDGs works  
2. A check mark is shown on the top right corner when SDGs are selected  
3. Deselected SDGs are shown in a darker color  
4. Form errors work correctly  

## Tracking

[LET-351](https://vizzuality.atlassian.net/browse/LET-351)

## Screenshots
<img width="1297" alt="Screenshot 2022-04-22 at 10 17 34" src="https://user-images.githubusercontent.com/6273795/164676488-f0275479-ff5d-4be8-9ce9-412fb7f39c25.png">
